include/triton/Dialect/TritonGPU/Transforms/Utility.h

```diff
@@ -169,6 +169,9 @@ Value linearize(OpBuilder &b, Location loc, ArrayRef<Value> multiDim,
 // operand and single result.
 bool isPureUnaryInlineAsm(Operation *op);
 
+// read the compute capability from the module attributes
+int getComputeCapability(Operation *module);
```

Comment:
This should ultimately be moved into NVidia specific utilities because the concept of a compute capability applies only to NVidia GPUs.  For the time being I guess we can leave it here but I would perhaps rename the function to `getNVidiaComputeCapability`

Reply:
Sure. Yes, I do see that as a possibly difficult thing to merge.